app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -836,6 +877,23 @@ export default class Heatmap {
           this.options.onRowLabelClick(d.label, d3.event)
       );
 
+    rowLabelEnter
+      .append("line")
+      .attr("x1", 0)
+      .attr("x2", this.rowLabelsWidth)
+      .attr("y1", this.cell.height)
+      .attr("y2", this.cell.height)
+      .attr("class", cs.genusBorder)
+      .classed(cs.hideGenusBorder, (label, i, nodes) => {
+        const nextLabel = this.filteredRowLabels[i + 1];
+        if (nextLabel) {
+          return label.label.split(" ")[0] === nextLabel.label.split(" ")[0];
```

Comment:
nit: you might want to change the name of the first variable... `label.label` fills very confusing.

Reply:
Sorry this line was out of date. 

Anyway, the `label.label` name predates this change and is also in use by the AMR heatmap.